roles/bugziller/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Install python-bugzilla
+  package: name=python-bugzilla state=latest
+
+- name: Setup python-bugzilla
+  command: "bugzilla login {{bugzilla_user}} {{bugzilla_pass}}"
```

Comment:
We need to have them in the private repo to work.

Also, how do we handle 1) cookie expiration 2) password change ?

(2 is less critical, but 1 bother me a bit)

Also, do we need a specific doc for that, or we can assume that "this is a service account, follow usual bugzilla process for any change aka send mail to xxx" is enough ?

Reply:
It's already in the private repo, so we have that sorted. Don't we keep running ansible on all hosts every day?

Reply:
(Ah, it's not in the private repo. The change is ready but not pushed)

Reply:
Indeed, that's run nightly so this would work. For some reason, I tought that the bugzilla login command was run just if we had no cookie file, but since we run it each time, i guess that's ok.

Now, if bugzilla login do not create a cookie if one already exist, then we will have a problem.